backend/tools/work_with_db.py

```diff
@@ -0,0 +1,4 @@
+def get_psw_from_db(db, email):
```

Comment:
the first parameter here doesn't make sense. you pass a module here and call it "db". That is a bad name at least. Can you make import here instead?